internal/wrr/wrr.go

```diff
@@ -21,12 +21,11 @@ package wrr
 
 // WRR defines an interface that implements weighted round robin.
 type WRR interface {
-	// Add adds an item with weight to the WRR set.
-	//
-	// Add and Next need to be thread safe.
+	// Add adds an item with weight to the WRR set. All Add must be only called
+	// before any calls to Next().
 	Add(item any, weight int64)
 	// Next returns the next picked item.
 	//
-	// Add and Next need to be thread safe.
+	// Next needs to be thread safe.
```

Comment:
Possibly for clarity: "Add may not be called after any call to Next."

Reply:
Done.